CodingStyle

```diff
@@ -88,18 +88,24 @@ by section.
 
         if ( foo ) { // no
 
-   - Always use newline following if:
-
-        if (foo)
-          bar;         // okay, but discouraged...
+   - Always use newline following if, and use braces:
```

Comment:
Question here: this change is prescriptive rather than descriptive of existing practice. In general, should we make commits to bring extant code in line with new guidelines when we see them? Should the guidelines themselves make a suggestion about that?

Reply:
(And FWIW, *I* like braceless single-statements, but am totally fine with this: an oversight here caused Heartbleed, IIRC. OTOH, compilers now issue diagnostics for deceptive indent levels, but with this adopted there's /syntactic/ disambiguity, which I'm generally on-board with.)

Reply:
apply to new code, and old code when it is touched.

Reply:
I think I'm the one that fought for the braceless single statement if back in 2011, but i've crossed back over from the dark side since then!

Reply:
+1 to both!